submissions/dafen173/frogger-game/app.js

```diff
@@ -0,0 +1,115 @@
+// Enemies our player must avoid.........
```

Comment:
this file belongs to another task, could you please remove it

Reply:
Impossible to open and close the popup by keyboard. Could you please fix it?

Reply:
> Impossible to open and close the popup by keyboard. Could you please fix it?

seems to have corrected all the points you indicated
it's okay now?